BurnOutSharp/ProtectionType/Macrovision.SafeDisc.cs

```diff
@@ -59,7 +59,19 @@ internal string SafeDiscCheckPortableExecutable(string file, PortableExecutable
             bool stxt371Section = pex.ContainsSection("stxt371", exact: true);
             bool stxt774Section = pex.ContainsSection("stxt774", exact: true);
             if (stxt371Section || stxt774Section)
-                return $"SafeDisc 2+";
+            {
+                int entryPointIndex = pex.FindEntryPointSectionIndex();
+                var entryPointSectionHeader = pex.SectionNames[entryPointIndex];
```

Comment:
- Please use `string` instead of `var`
- This is not the header, rather the section name, so please rename to `entryPointSectionName` or something similar